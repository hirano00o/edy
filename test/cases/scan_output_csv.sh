#!/bin/bash

SCRIPT_ROOT_DIR=$1
TEST_NAME=$(basename "$0" | sed "s/\..*//")

# aws dynamodb scan --table-name User \
#   --filter-expression "ID = :id1 or ID = :id2" \
#   --expression-attribute-values "{\":id1\":{\"N\":\"12\"}, \":id2\":{\"N\":\"13\"}}" \
#   --endpoint-url http://localhost:8000
CMD="edy s -t User -f \"ID,N = 12 or ID,N = 13\" -o csv --local 8000"

. "${SCRIPT_ROOT_DIR}"/helper.sh

run_such_query_helper
