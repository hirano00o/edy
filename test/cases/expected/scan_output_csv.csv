Age,Birthday,Email,ID,Name
32,map[Day:30 Month:3 Year:1989],ivan@example.com,12,Ivan
32,map[Day:28 Month:2 Year:1989],justin@example.com,13,Justin
